lib/Mojo/WebSocketProxy/RequestLogger.pm

```diff
@@ -0,0 +1,91 @@
+use Object::Pad;
+
+class Mojo::WebSocketProxy::RequestLogger;
+
+use strict;
+use warnings;
+use Log::Any qw($log);
+use UUID::Tiny;
+
+field $context;
+
+our $default_handler = sub {
```

Comment:
IMO, When we call it, it'll be slightly misleading. maybe `log_handler` is a better name.

Reply:
agreed! will change it